portality/models/bibjson.py

```diff
@@ -101,7 +101,10 @@ def add_keyword(self, keyword):
         self._add_to_list_with_struct("keywords", keyword)
 
     def set_keywords(self, keywords):
-        self._set_with_struct("keywords", keywords)
+        lowercase_keywords = []
+        for k in keywords:
+            lowercase_keywords.append(k.lower())
+        self._set_with_struct("keywords", lowercase_keywords)
```

Comment:
Can we modify one of our existing unit tests to demonstrate this working.  Probably `test_models.py` would be the right one.